packages/cmf/src/cmfConnect.js

```diff
@@ -111,7 +111,7 @@ export function getStateToProps({
 
 	let userProps = {};
 	if (mapStateToProps) {
-		userProps = mapStateToProps(state, ownProps, cmfProps);
+		userProps = mapStateToProps(state, { ...ownProps, ...viewProps }, cmfProps);
```

Comment:
instead { ...ownProps, ...viewProps }, use rather { ...viewProps, ...ownProps } to allow to override the values by ownProps.